sdk/program/src/lib.rs

```diff
@@ -183,7 +183,6 @@
 //! [ed25519]: https://ed25519.cr.yp.to/
 //! [`Keypair`]: https://docs.rs/solana-sdk/latest/solana_sdk/signer/keypair/struct.Keypair.html
 //! [SHA-256]: https://en.wikipedia.org/wiki/SHA-2
-//! [`Sol`]: native_token::Sol
```

Comment:
Shouldn't this still work thanks to the re-export? If not, we should remove all of the other links here too (can be done in another PR though)